sdk/storage/storage-blob/src/StorageClient.ts

```diff
@@ -19,13 +19,6 @@ export interface CommonOptions {
   tracingOptions?: OperationTracingOptions;
 }
 
-export interface OperationTracingOptions {
```

Comment:
Do we want to re-export it here so it's not  a breaking change?

Reply:
Not here, re-export from index

Reply:
never mind, I added the export earlier this month in index.ts so it hasn't been released yet.